f8a_report/report_helper.py

```diff
@@ -684,6 +686,10 @@ def get_report(self, start_date, end_date, frequency='daily'):
             else:
                 ingestion_results = False
 
+            result = self.sentry_helper.retrieve_sentry_logs(start_date, end_date)
+            if result == {}:
```

Comment:
if not result: is more pythonic

Reply:
changed accordingly. thanks for suggestion